src/pybind/mgr/dashboard/frontend/src/app/shared/models/host-schema.ts

```diff
@@ -0,0 +1,14 @@
+export interface Host {
+  addr: string;
+  ceph_version: string;
+  hostname: string;
+  labels: string[];
+  services: string[];
+  sources: {
+    ceph: boolean;
+    orchestrator: boolean;
+  };
+  status: string;
+}
+
+export type HostList = Host[];
```

Comment:
thanks. shall we call the file `host.model.ts` to follow our other files in the folder?